account_invoice_pricelist_discount/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+This module extends OCA account_invoice_pricelist module behaviour
+
+It show public price & discount to the customer in invoice lines according to the the pricelist selected.
```

Comment:
It adds discount directly into the "discount" filed of the  invoice lines based on the the pricelist selected

Reply:
Fixed